examples/all-clusters-app/nrfconnect/sysbuild.conf

```diff
@@ -0,0 +1,18 @@
+#
+#    Copyright (c) 2024 Project CHIP Authors
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+
+SB_CONFIG_MATTER=y
+SB_CONFIG_MATTER_OTA=n
```

Comment:
Should it be 'n' even when using `prj_dfu.conf`?

Reply:
No, you are right, I've added sysbuild_dfu and _release.confs to set it to y.